packages/dashboard/src/components/schedule-visualizer/index.tsx

```diff
@@ -44,7 +43,7 @@ export interface MapFloorLayer {
 export interface ScheduleVisualizerProps extends React.PropsWithChildren<{}> {
   buildingMap: RomiCore.BuildingMap;
   negotiationTrajStore: Readonly<Record<string, NegotiationTrajectoryResponse>>;
-  omniPanelView: number;
+  showTrajectories: boolean;
```

Comment:
Suggest to make this optional and default to `true`.

Reply:
set it to optional

Added a check for showTrajectories === undefined which will result in the default behaviour when showTrajectory is true